lib/response.js

```diff
@@ -316,17 +316,23 @@ function actionPDF (req, res, note) {
   var content = extracted.markdown
   var title = models.Note.decodeTitle(note.title)
 
+  var appDir = path.dirname(require.main.filename)
+  var highlightCssPath = appDir + '/lib/hljs.css'
```

Comment:
We put css assets under `public/css` directory as convention. Could you add one config above:

https://github.com/hackmdio/codimd/blob/56cc64e9109f1160d66a805ec693f07751fc5c1a/lib/config/index.js#L203

```js
config.appRootPath = appRootPath
```

So we can find `highlightCssPath` here by:

```js
var highlightCssPath = path.join(config.appRootPath, 'public/css/hljs.css')
```

Reply:
Seems like `hljs.css` is taken from `/node_modules/highlight.js/styles/default.css`, we can use:

```js
var highlightCssPath = path.join(config.appRootPath, '/node_modules/highlight.js/styles/default.css')
```

(however the minification is lost 😅 )

If we make highlight.js theme configurable in the future, this can be done easier.

Reply:
I fixed them as you pointed out :+1: 
(I set not `default.css` but `github-gist.css` to `highlightCssPath` because View page uses this theme!)